docs/development/compare_atomic_files.ipynb

```diff
@@ -6,145 +6,8878 @@
    "source": [
```

Comment:
<p>Is this code still needed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/carsus/pull/315/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
I do not think so. The code is pretty outdated and doesn't work for some key values. I would remove it once I am sure `AtomDataCompare` also does all the things this class does.